xAPI.md

```diff
@@ -255,6 +255,25 @@ learning that SCORM could not enable.
 
 ## 1.0 Summary of the Experience API
 
+The Experience API (Application Programming Interface) - or xAPI as it is commonly called - is a framework that 
+facilitates the description of learning experiences and their communication and storage in a consistent manner 
+in a digital environment. As an application programming interface, the xAPI does not set out how the framework  
+should be implemented. Rather, it sets out the essential components of that framework and how they relate to each other. The purpose of the xAPI and the goals of this document are:
+
+* To maximize interoperability and reliability of computer programs which create, gather and store information  
```

Comment:
I think the emphasis on computer programs and programmers is unhelpful. Certainly programs written by programmers will be on the front line of interoperability, but the goal is interoperability of the wider systems that these programs are part of.